crates/store/re_video/src/decode/ffmpeg.rs

```diff
@@ -362,17 +383,9 @@ fn read_ffmpeg_output(
                 }
             }
 
-            FfmpegEvent::Log(LogLevel::Warning, mut msg) => {
+            FfmpegEvent::Log(LogLevel::Warning, msg) => {
                 if !should_ignore_log_msg(&msg) {
-                    // Make warn_once work on `[swscaler @ 0x148db8000]` style warnings even if the address is different every time.
-                    if let Some(pos) = msg.find("[swscaler @ 0x") {
-                        msg = [
-                            &msg[..pos],
-                            &msg[(pos + "[swscaler @ 0x148db8000]".len())..],
-                        ]
-                        .join("[swscaler]");
-                    };
-                    re_log::warn_once!("{debug_name} decoder: {msg}");
+                    log_ffmpeg_warning_once(debug_name, &msg);
```

Comment:
If we had a helper-function this could be:

```rs
let msg = sanitize_message(msg);
if !should_ignore_log_msg(&msg) {
    re_log::warn_once!("{debug_name} decoder: {msg}");
}
```